rustls/src/crypto/ring/ticketer.rs

```diff
@@ -40,7 +40,7 @@ impl Ticketer {
     pub fn new<M: crate::lock::MakeMutex>(
         time_provider: &'static dyn TimeProvider,
     ) -> Result<Arc<dyn ProducesTickets>, Error> {
-        Ok(Arc::new(crate::ticketer::TicketSwitcher::new::<M>(
+        Ok(Arc::new(crate::ticketer::TicketRotator::new::<M>(
```

Comment:
This is very strange; I don't think this actually will compile. And it doesn't on `main` either, not since this code was first introduced back for 0.23.5!

Which is mainly a concern because this PR's CI is green it that shows we don't have any coverage of `cargo build --lib --no-default-features --features ring,hashbrown`.

I suggest we delete this entire function, and the matching one in `aws_lc_rs/ticketer.rs`. And then alter the feature gates on these files from `#[cfg(any(feature = "std", feature = "hashbrown"))]` to just `#[cfg(feature = "std")]`

Reply:
I suggest: `git cherry-pick 6584005e8c22979548daac188d78abf57aa2a71b` and then using `git rebase -i` to put that commit above this one.